### ISSUE TITLE
ICCV 2023 | SLCA: Slow Learner with Classifier Alignment for Continual Learning on a Pre-trained Model

### ISSUE THREAD

**Issue by chufanchen** [chufanchen/read-paper-and-code#158]
https://github.com/GengDavid/SLCA

https://arxiv.org/abs/2303.05118

**Comment by ChufanSuki** [comment]
### continual learning on a pre-trained model(CLPM)

There are two major strategies to leverage the pre-trained knowledge

1. fine-tune the representations
2. keep the representations fixed while learning a few additional parameters

**Comment by ChufanSuki** [comment]
### Slow learner

using a much smaller learning rate ($0.0001$) for $θ_{rps}$ and a slightly larger learning rate ($0.01$) for $θ_{cls}$ can greatly enhance the performance of traditional baseline

The proposed SL can almost address the problem of the representation layer,
yet the classification layer remains sub-optimal
